### PR TITLE
Add CreatorID to stream and asset

### PR DESCRIPTION
**Pull request by leszko**

### CODE REVIEW

File: api.go [hunk 1]
```diff
@@ -208,7 +208,8 @@ type (
 		RecordingURL    string `json:"recordingUrl,omitempty"`
 		Mp4Url          string `json:"mp4Url,omitempty"`
 
-		Pull *StreamPull `json:"pull,omitempty"`
+		Pull      *StreamPull `json:"pull,omitempty"`
+		CreatorID CreatorID   `json:"creatorID,omitempty"`
```
Comment (victorges) [inline api.go:212]: I'd keep this as a pointer since it's an optional field (omitempty is noop here)

File: api.go [hunk 2]
```diff
@@ -438,7 +444,8 @@ type (
 		Source        AssetSource `json:"source"`
 		Status        AssetStatus `json:"status"`
 		AssetSpec
-		SourcePlaybackReady bool `json:"sourcePlaybackReady"`
+		SourcePlaybackReady bool      `json:"sourcePlaybackReady"`
+		CreatorID           CreatorID `json:"creatorId,omitempty"`
```
Comment (victorges) [inline api.go:448]: same

**Review by victorges — APPROVED** [review]

File: api.go (hunk 2 above)
Comment (leszko) [reply api.go]: Thanks, updated!

File: api.go (hunk 1 above)
Comment (leszko) [reply api.go]: Thanks, updated!

**Pull request merged**